.editorconfig

```diff
@@ -10,3 +10,7 @@ indent_style = space
 indent_size = 4
 insert_final_newline = true
 trim_trailing_whitespace = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
I see 2 spaces being used almost everywhere for json:

https://docs.npmjs.com/misc/coding-style

For example, `bower.json` is 2 spaces as well. 

I noticed noticed now composer init creates it with 4 spaces, not sure why though.